functions/delete-many/1.0/index.js

```diff
@@ -0,0 +1,34 @@
+const deleteMany = async ({ collection }) => {
+  try {
+    const {
+      data,
+      model: { name: modelName },
+    } = collection;
+    const ids = data.map((item) => item.id);
+    const count = ids.length;
+    const mutation = `
+        mutation {
+          deleteMany${modelName}(input: $input) {
+            id
+          }
+        }
+      `;
+
+    const { errors } = await gql(mutation, { input: { ids } });
+    if (errors) {
+      throw errors;
+    }
+
+    return {
+      result: `${count} ${modelName} records have been deleted`,
+    };
+  } catch (error) {
+    if (error instanceof TypeError) {
+      throw new Error('One record could not be found');
```

Comment:
Does a `TypeError` always mean a record could not be found?

Reply:
I copied and pasted this from the native delete step, but I reckon this could have multiple different causes. Will fix this by just throwing the actual error.